css/style.css

```diff
@@ -45,11 +45,11 @@ header ul li a{
 }
 header{
     display: inline;
-    background-color: #2874f0;
+    background-color: #17de04;
```

Comment:
this is not needed in this code because there is no effect of this in the main branch